src/malloc_wrapper.cc

```diff
@@ -46,7 +46,7 @@ void gnu_backtrace()
 	}
 
 	for (j = 0; j < nptrs; j++)
-		printf("%s\n", strings[j]);
```

Comment:
[google-readability-braces-around-statements]: statement should be inside braces